exonum-java-binding/package_app.sh

```diff
@@ -44,6 +44,19 @@ function build-exonum-java-for-platform() {
 }
 
 function build-exonum-java-macos() {
+    # We use static linkage for RocksDB on Mac because we depend on RocksDB 5.18.3
+    # which is not available via Homebrew
+    export ROCKSDB_STATIC=1
+    # Check if ROCKSDB_LIB_DIR is set
+    if [ -z "${ROCKSDB_LIB_DIR:-}" ]; then
```

Comment:
why do we need these variables? It's always the same `/usr/local/lib`. Can we set them here instead of requiring that from users?

Reply:
We're not requiring this from users, because our users will use the Homebrew package :wink:
Setting this variable explicitly will allow you to choose what RocksDB version to use. Besides, it is also mentioned in [Exonum install guide](https://exonum.com/doc/version/latest/get-started/install/)

Reply:
Mb it'll be better to replace by the following (in pseudo-code):
```
export ROCKSDB_LIB_DIR={ROCKSDB_LIB_DIR:-/usr/local/lib}
IF $ROCKSDB_LIB_DIR/librocksdb.dylib not found THEN fail(RocksDB not installed)
```

Reply:
actually, you can use a specific version of the rocksdb and use general path for snappy:
```
export ROCKSDB_LIB_DIR=/usr/local/Cellar/rocksdb/6.1.2/lib
or
export ROCKSDB_LIB_DIR=/usr/local/Cellar/rocksdb5/5.18.3/lib
and
export SNAPPY_LIB_DIR=/usr/local/lib
```